ios/FluentUI/PreviewCard/PreviewCardTokens.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+/// Pre-defined sizes of the PreviewCard.
+@objc public enum MSFPreviewCardSize: Int, CaseIterable {
+    case ultrawideHeight
+    case landscapeHeight
+    case oldLandscapeHeight
+    case squareHeight
+    var cardHeight: CGFloat {
+        switch self {
+        case .ultrawideHeight:
+        return 147
```

Comment:
I assume these are just based on the aspect ratio and minWidth, but it might be worth adding a comment to explain where these numbers come from.

Reply:
Yup, only the height of the cards since the width never changes. I'll add a comment, good note!

Reply:
where is the terminology "oldlandscape" come from?

Reply:
Old landscape was the standard aspect ratio before modern TVs switched to 16:9 with the rise of HD. Certain devices still use 4:3 like Apple's iPad

Reply:
Is the term "old landscape" commonly used elsewhere?

Reply:
No idea. Old landscape popped up as 4:3 a lot while I researched it, but I now believe fullscreen would be more appropriate. Apparently, that's the official name for 4:3 aspect ratio